web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -287,7 +287,11 @@ function ProjectTopSection({
             )}
             // TODO if no off-chain data, use on-chain project.issuer
             issuer={getParty(project?.partyByIssuerId)}
-            reseller={getParty(project?.partyByResellerId)}
+            reseller={
+              !isVCSProject &&
+              project?.partyByResellerId &&
+              getParty(project?.partyByResellerId)
+            }
```

Comment:
I think this field is supposed to receive either `User` or `undefined`. Not sure why TS is not screaming now that it can receive a boolean but we should probably send undefined when the condition is false.

Reply:
will review, thanks!